lua/terrortown/lang/en.lua

```diff
@@ -1790,19 +1791,19 @@ L.label_doors_force_pairs = "Force close-by doors as double doors"
 L.label_doors_destructible = "Enable destructible doors"
 L.label_doors_locked_indestructible = "Initially locked doors are indestructible"
 L.label_doors_health = "Door health"
-L.label_doors_prop_health = "Door prop health"
+L.label_doors_prop_health = "Destructed door health"
```

Comment:
This doesn't sound right, but i would have to check where exactly this string is used

Reply:
This is a destructable door.
![Desktop Screenshot 2022 12 30 - 17 00 31 75](https://user-images.githubusercontent.com/23385240/210078248-46ab0678-7a74-4d2d-8239-9dfe8959d5b8.png)
And this is a destructed door (that you can break) after the destructable door was... destructed. Yeah, I know how that sounds. 😅
![Desktop Screenshot 2022 12 30 - 17 00 40 50](https://user-images.githubusercontent.com/23385240/210078429-f3911fe9-1e21-45cb-a1bd-dea287fed813.png)

Reply:
Ahh i see, now i know why the '_prop_' in the identifier :D

Your translation makes that more clear actually. Thanks!